manifests/init.pp

```diff
@@ -5,6 +5,43 @@
 #
 # @example
 #   include pe_event_forwarding
+#
+# @param [Optional[String]] pe_username
+#   PE username
+# @param [Optional[Sensitive[String]]] pe_password
+#   PE password
+# @param [Optional[String]] pe_token
+#   PE token
+# @param [Optional[String]] pe_console
+#   PE console
+# @param [Optional[Boolean]] disabled
+#   When true, removes cron job
+# @param [Optional[String]] cron_minute
+#   Sets cron minute (0-59)
+# @param [Optional[String]] cron_hour
+#   Sets cron hour (0-23)
+# @param [Optional[String]] cron_weekday
+#   Sets cron day of the week (0-6)
+# @param [Optional[String]] cron_month
+#   Sets cron month (1-12)
+# @param [Optional[String]] cron_monthday
+#   Sets cron day of the month (1-31)
+# @param [Optional[String]] log_path
+#   Path to desired location for log files
+# @param [Optional[String]] lock_path
+#   Path to desired location for lock file
```

Comment:
Same for this parameter. It should receive a directory, to which the rest of the path `/pe_event_forwarding/cache/state/events_collection_run.lock` will be appended.